src/components/organisms/Settings/Account/hooks.ts

```diff
@@ -2,9 +2,21 @@ import { useApolloClient } from "@apollo/client";
 import { useCallback } from "react";
 import { useIntl } from "react-intl";
 
-import { useUpdateMeMutation, useProfileQuery } from "@reearth/gql";
+import { useUpdateMeMutation, useProfileQuery, Theme as GQLTheme } from "@reearth/gql";
 import { useTeam, useProject, useNotification } from "@reearth/state";
-import { toGQLEnum } from "@reearth/util/value";
+
+const enumTypeMapper: Partial<Record<GQLTheme, string>> = {
+  [GQLTheme.Default]: "default",
+  [GQLTheme.Dark]: "dark",
+  [GQLTheme.Light]: "light",
+};
+
+export type Theme = "dark" | "light";
```

Comment:
The types are not consistent. enumTypeMapper contains "default", but Theme type does not. it would be better to add "default" to Theme type.

```suggestion
export type Theme = "dark" | "light" | "default";
```